articles/errors/deprecation-errors.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Deprecation Error Reference
+description: A listing of errors and descriptions relating to deprecations.
+toc: true
+---
+# Deprecation Error Reference
+
+## How to search logs for deprecation warnings
+
+There are two different ways to search for warning messages showing usage of deprecated features: The Auth0 Dashboard or the Auth0 Management API. Note that in either case, the [log retention period](/logs#how-long-is-log-file-data-available-) is governed by the subscription level of your account.
+
+### Search logs via the Dashboard
+
+In the [Auth0 Dashboard](/${manage_url}), if any deprecated features are being used, a "Deprecation Notice" message will show up in the **Logs** section of the dashboard. 
```

Comment:
just `Dashboard`. And reverse the this sentence so it's more easily readable:
If you use any deprecated feature, a `Deprecation Notice` message will show up in the **Logs** section of the [Dashboard](/${manage_url}).